### ISSUE TITLE
EventBusService.createCronJob broken

### ISSUE THREAD

**Issue by dwene** [medusajs/medusa#2886]
With the latest Medusa version, it appears that the `EventBusService` is missing assignment of this.jobSchedulerService in the constructor. This breaks the `EventBusService.createCronJob` call.

```
TypeError: Cannot read properties of undefined (reading 'create')
    at EventBusService.createCronJob (/Users/derekwene/src/lambdacurry/markethaus-starter/medusa-multivendor-api/node_modules/@medusajs/medusa/src/services/event-bus.ts:283:31)
```

https://github.com/medusajs/medusa/blob/master/packages/medusa/src/services/event-bus.ts#L39

### System information

Medusa version (including plugins): 1.7.1
Node.js version: 16
Database: PG
Operating system: OSX M1
Browser (if relevant):

### Steps to reproduce the behavior

Try to add eventBusService.createCronJob in a subscriber.

See stack error on startup.

### Expected behavior

Even though eventBusService.createCronJob is deprecated, it should still work until it is removed :)

**Comment by srindom** [comment]
Thanks for reporting - we will fix this ASAP. 

In any case, I'd suggest that you move to the new `JobSchedulerService` :)

**Comment by olivermrbl** [comment]
Released in [v1.7.2](https://github.com/medusajs/medusa/releases/tag/v1.7.1)
